File: utils/VectorEnginePlugin.ref/pares/funciones/primSpanLeft.c
/* VectorEnginePlugin>>#spanLeft */
EXPORT(sqInt)
primSpanLeft(void)
{
	/* (int(z+1)) works equally well than the more intuitive but slower (int(ceil(z)) */
	methodReturnValue(integerObjectOf((((sqInt)(((spanLeft - auxStrokeWidthDilatedHalf) - subPixelDelta) + 1)))));
	return null;
}

File: utils/VectorEnginePlugin.ref/pares/funciones/primSpanLeft.js
function primSpanLeft(argCount) {
    // (int(z+1)) works equally well than the more intuitive but slower (int(ceil(z))
    methodReturnValue(integerObjectOf(Math.trunc(((spanLeft - auxStrokeWidthDilatedHalf) - subPixelDelta) + 1)));
    return true;
}
